docs/api/index.md

```diff
@@ -132,7 +132,9 @@ issues as the SSQP protocol demands case insensitive lookups.
 - `type`: `"s3"`
 - `bucket`: the name of the S3 bucket
 - `prefix`: a path prefix to put in front of all keys (eg: `/windows`)
-- `region`: the AWS region where the bucket is located
+- `region`: the AWS region where the bucket is located. In order to use a
+  custom region, for example to use an S3 compatible service such as Ceph or
+  minio, specify a tuple like: `["custom-region-name", "http://minio-address/"]`.
```

Comment:
maybe also give the simple sting example for non-custom regions?

Reply:
good idea, done!